emu/src/arm/psr.rs

```diff
@@ -2,7 +2,7 @@ use crate::arm::{condition::Condition, cpu_modes::Mode, data_processing::Arithme
 use crate::bitwise::Bits;
 
 /// Program Status Register.
-#[derive(Default)]
+#[derive(Default, Clone)]
```

Comment:
We can also add `Copy` since is a built in type, no wasted copy

Reply:
Sure, thanks!